pkg/controller/download_controller.go

```diff
@@ -82,7 +82,7 @@ func (c *downloadController) enqueueDownload(obj interface{}) {
 		return
 	}
 
-	log.Debug("Enqueueing Download")
+	log.Info("Enqueueing Download")
```

Comment:
Which object are we enqueueing the download for?

Reply:
The same as above.